src/sections/About/about.style.js

```diff
@@ -20,7 +20,22 @@ const AboutSectionWrapper = styled.section`
       left: -22px;
```

Comment:
@leecalcote .....line: 4 in this file `padding: 120px 0 100px 0` is causing that empty white space on the page, if you could change it to `margin: 1rem 0 5rem` that would do the trick...

Reply:
Thank you, @Nikhil-Ladha. I have updated the spacing